services/notifications.go

```diff
@@ -639,21 +639,6 @@ func queueEmailNotifications(notificationsByUserID map[uint64]map[types.EventNam
 			continue
 		}
 		go func(userEmail string, userNotifications map[types.EventName][]types.Notification) {
-			notification := ""
-			othernotifications := ""
-			i := 0
-			for notificationEvent := range userNotifications {
-				if i == 0 {
-					notification = string(notificationEvent)
-				} else if i == 1 {
-					othernotifications = fmt.Sprintf(" and %s", notificationEvent)
```

Comment:
While working on this I also fixed two other small bugs.
The first here, this line is never used since you check for `i > 1` below it and overwrite `othernotifications`.